fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,38 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+ReferenceReg=`opts_GetOpt1 "--refreg" $@`                            # How to register bold image to reference? Using 'linear' (default) or also 'nonlinear' registration to movement reference.
```

Comment:
I don't see edits adding this to the help info.  Also, the sub-script also tests for if being `NONE` at one point...

Reply:
In `GenericfMRIVolumeProcessingPipeline.sh`:

```
show_usage() {
    echo "Usage information To Be Written"
    exit 1
}
```

But I added the info to `OneStepResampling.sh`. This parameter is in interaction with the BOLD reference PR. Namely, the parameter is only relevant if another BOLD reference is in use. If it isn't, then `refreg` is set to NONE. See line 338 in PR: <https://github.com/Washington-University/HCPpipelines/pull/140>

```
ReferenceReg="NONE"  # only do nonlinear or linear registration for target that is not of the same bold run
```